test/e2e/README.md

```diff
@@ -0,0 +1,16 @@
+# E2E Testing
+
+End-to-end (e2e) testing is automated testing for real user scenarios.
+
+## Build and run test
+
+Prerequisites:
+- a running k8s cluster and kube config. We will need to pass kube config as arguments.
+- Have kubeconfig file ready.
+
+e2e tests are written as Go test. All go test techniques apply, e.g. picking
+what to run, timeout length. Let's say I want to run all tests in "test/e2e/":
+
+```
+$ go test -v ./test/e2e/ --kubeconfig "$HOME/.kube/config" --gameserver-image=gcr.io/agones-images/cpp-simple-server:0.2
```

Comment:
Should we be running tests this way, or through `make e2e` ?

Reply:
you should use `make e2e`, I will change.